src/main/java/net/sf/rails/game/MapOrientation.java

```diff
@@ -126,15 +126,15 @@ public String getUIClassName() {
         default:
             throw new AssertionError(this);
         }
-        
+
     }
 
     // information to define neighbours
     private static final int[] rowDeltaNS =
             new int[] { +2, +1, -1, -2, -1, +1 };
-    private static final int[] colDeltaNS = 
+    private static final int[] colDeltaNS =
             new int[] {  0, -1, -1,  0, +1, +1 };
-    private static final int[] rowDeltaEW = 
+    private static final int[] rowDeltaEW =
             new int[] { +1,  0, -1, -1,  0, +1 };
     private static final int[] colDeltaEW =
             new int[] { -1, -2, -1, +1, +2, +1 };
```

Comment:
Perhaps change to single line definitions? I.e.:

```java
    private static final int[] rowDeltaNS = new int[] { +2, +1, -1, -2, -1, +1 };
    private static final int[] colDeltaNS = new int[] {  0, -1, -1,  0, +1, +1 };
    private static final int[] rowDeltaEW = new int[] { +1,  0, -1, -1,  0, +1 };
    private static final int[] colDeltaEW = new int[] { -1, -2, -1, +1, +2, +1 };
```

Reply:
Done-- will commit as part of the next chunk of code cleanup